From 249ba2227493af896ebe408a2ad4866faf8e8e81 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Tue, 15 Sep 2015 13:08:40 +0200
Subject: [PATCH] Install LibMagic.pm from lib only
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This prevents from installing LibMagic.xs and typemap.

CPAN RT#107081

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 Makefile.PL | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile.PL b/Makefile.PL
index d306f8e..62723e8 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -27,6 +27,9 @@ my %WriteMakefileArgs = (
   "MIN_PERL_VERSION" => "5.008",
   "NAME" => "File::LibMagic",
   "OBJECT" => "lib/File/LibMagic\$(OBJ_EXT)",
+  "PM" => {
+    "lib/File/LibMagic.pm" => "\$(INST_LIB)/File/LibMagic.pm",
+  },
   "PREREQ_PM" => {
     "Carp" => 0,
     "Exporter" => 0,
-- 
2.4.3

